src/test/codegen/no-plt.rs

```diff
@@ -1,3 +1,4 @@
+// no-system-llvm
```

Comment:
Again, this looks suspicious to me. Are you sure this is caused by the alloc patch?

FileCheck indicates that this line is now `Function Attrs: nofree nounwind nonlazybind readonly uwtable`. I'd assume that's because `nofree` is newly inferred, or it gets matched against the wrong function.

Reply:
(Same as the other two cases.)